Makefile

```diff
@@ -1,21 +1,26 @@
 NAME=timescaledb
 ORG=timescale
+PG_VER=pg10
 VERSION=$(shell awk '/^ENV TIMESCALEDB_VERSION/ {print $$3}' Dockerfile)
 
 default: image
 
-.build_$(VERSION): Dockerfile backup_init.sh
-	docker build -t $(ORG)/$(NAME) .
-	docker tag $(ORG)/$(NAME):latest $(ORG)/$(NAME):$(VERSION)
-	touch .build_$(VERSION)
+.build_$(VERSION)_$(PG_VER): Dockerfile backup_init.sh
+ifeq ($(PG_VER),pg9.6)
+	docker build --build-arg PG_VERSION=9.6.6 -t $(ORG)/$(NAME):latest-$(PG_VER) .
+else
+	docker build --build-arg PG_VERSION=10.1 -t $(ORG)/$(NAME):latest-$(PG_VER) .
+endif
+	docker tag $(ORG)/$(NAME):latest-$(PG_VER) $(ORG)/$(NAME):$(VERSION)-$(PG_VER)
```

Comment:
Don't we want a tag `:$(VERSION)` that defaults to a particular PG version (like the latest/pg10)?

Reply:
I'd rather we have explicit tags so people know which version of PostgreSQL it is based on.

Reply:
I thought we decided to have `latest` tracking latest PG release. Also, having less specific tags does not exclude more explicit tags. In fact, it is pretty standard to have multiple tags. Postgres, for instance, has `latest`, `10` and `10.1` all pointing to the same image.